### PR TITLE
Conan release 1.44

### PR DESCRIPTION
**Pull request by czoido**

### CODE REVIEW

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+```
+
+- **Remove broken symlinks**, no matter if relative or absolute.
+
+```python
+def remove_broken_symlinks(conanfile, base_folder):
+```
+
+These tools will help migrate to Conan 2.0 where the package files won't be automatically
+cleaned from broken absolute symlinks or external symlinks.
```
Comment (lasote) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:90]: I would move these lines to line 67

**Review by lasote — APPROVED** [review]

**Review by memsharded — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
Screenshot 2022-01-17 at 13-41-16 Conan 1 44 Introduced tool_requires attribute to replace generic build_requires, new tool

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    }
+},
+ "datePublished": "2022-01-17",
+ "description": "New tool_requires attribute, explicitly handle symlinks, several user toolchains in CMakeToolchain, added vars to CMakeDeps.",
+ }
+</script>
+
+
+We are pleased to announce that Conan 1.44 has been released and brings some significant
+new features and bug fixes. We have added a
```
Comment (tapia) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:33]:
```suggestion
new features and bug fixes. We added a
```

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+added to
+[CMakeToolchain](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmaketoolchain.html)
+the capability of applying several user toolchains. Finally,
+[CMakeDeps](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmakedeps.html)
+adds several variables to match upstream CMake modules as much as possible.
+
+
+## New tool_requires attribute
+
+Starting in Conan 1.44, we have renamed “build requirement” to "tool requirement”. The
```
Comment (tapia) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:46]:
```suggestion
Starting in Conan 1.44, we renamed “build requirement” to "tool requirement”. The
```

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+intention of this is to highlight that the usage of this kind of requirement must be for
```
Comment (tapia) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:47]:
```suggestion
purpose of this is to highlight that the usage of this kind of requirement must be for
```

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+to make them ready for 2.0.
+
+## New tools to explicitly handle symlinks
+
+There are [several new
+functions](https://docs.conan.io/en/latest/reference/conanfile/tools/files/symlinks.html)
+under `conan.tools.files.symlinks` to manage symlinks. With these tools, you can transform
+absolute to relative symlinks, remove broken symlinks, remove external symlinks and get
+the symlinks in a folder. These tools will help migrate to Conan 2.0 where the package
+files won't be automatically cleaned from broken absolute symlinks or external symlinks.
```
Comment (tapia) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:68]: This sentence is a little bit confusing. Could you rephrase it, please?

File: _posts/2022-01-17-New-conan-release-1-44.markdown [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+include("/path/to/mytoolchain.cmake")
+include("/otherpath/to/mytoolchain.cmake")
+...
+```
+
+## Added variables to CMakeDeps to match better with upstream
+
+As a way of getting closer to the typical variables declared by [CMake official find
+modules](https://cmake.org/cmake/help/latest/manual/cmake-modules.7.html#find-modules) we
+have added these variables so they are available when using
```
Comment (tapia) [inline _posts/2022-01-17-New-conan-release-1-44.markdown:147]:
```suggestion
added these variables so they are available when using
```
